Clinical-Genomics/scout
# Bulk dismiss SNVs, SVs and cancer SNVs

Fix #2081 

This PR:
- removes the code to dismiss single variants from variants pages (all types of variants).
- Adds a system to dismiss many variants at the time for 
  - SNVs (rare disease track)
  - SVs (rare disease track)
  - Cancer SNVs (cancer track)

Screenshot:
![image](https://user-images.githubusercontent.com/28093618/92728235-ac404000-f370-11ea-8756-c7d56eb2aa0a.png)


I've tried to apply this system also to cancer SVs and STRS but I had problems:
- cancer SVs  returns an error that I couldn't fix yet 
- and STRS page contains a form to show bam files and since this multi-checkbox system must be included in another form, but you can't have a form inside a form, I just reverted the code to how it was before the dismiss.

Of course the things above might be fixed, but it will take time, and we don't have much time for the patch of current version..


**How to test**:
1. On stage, try to dismiss SNVs, SVs and cancer SNVs.
1. Make sure that if you were filtering the variants using any filter, these filters are still used when the page reloads after dismissing one or more variants
1. It shouldn't be possible to dismiss single variants any more from STRs and cancer SVs (cancer SVs behavior might be tested on a local instance of scout after loading the demo cancer sample)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
